templates/back-office/mobile-notification-content.hypr

```diff
@@ -0,0 +1,107 @@
+<div class="header shipment-Info-header">
```

Comment:
Please format the HTML in this document. The tabbing is not consistent and some extra new lines can be removed.